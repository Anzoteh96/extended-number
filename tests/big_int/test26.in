-4470466589875808974031275432302589302048943813098129917305073734932645906678239
592577825244819603538031532732186821951663811212735395723576731645638417591537
