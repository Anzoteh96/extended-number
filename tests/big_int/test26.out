Enter two numbers n and m:
n is -4470466589875808974031275432302589302048943813098129917305073734932645906678239
m is 592577825244819603538031532732186821951663811212735395723576731645638417591537
n + m is -3877888764630989370493243899570402480097280001885394521581497003287007489086702
n - m is -5063044415120628577569306965034776124000607624310865313028650466578284324269776
n * m is -2649099369658231760110346041354891188516060424187934235958160809751201567402782651459511707028087141360979847409856127535498931087258751206829518006388463343
n / m is -8
n % m is 270156012082747854272976829554905273564366676603753248483540118232461434054057
